lib/node_modules/@stdlib/math/base/tools/evalrational-compile-c/lib/main.js

```diff
@@ -53,14 +53,15 @@ var NAN_TEMPLATE = readFile( join( dir, 'nan.c.txt' ), opts );
 * @returns {string} serialized value
 */
 function value2string( x ) {
-	var str = x.toString();
+	var str;
 	if ( x === PINF ) {
```

Comment:
```JavaScript
if ( x === PINF ) {
        // eslint-disable-next-line no-useless-concat
        return '1.0' + '{{dtype_suffix}}' + ' / 0.0' + '{{dtype_suffix}}';
}
if ( x === NINF ) {
        // eslint-disable-next-line no-useless-concat
        return '-1.0' + '{{dtype_suffix}}' + ' / 0.0' + '{{dtype_suffix}}';
}
```

Can we use `{{dtype_suffix}}` in single quotes here, to avoid getting Lint errors ?
cc: @kgryte